catalog/plugin/catalog-plugin-duplication/src/main/java/ddf/catalog/metacard/duplication/DuplicationValidator.java

```diff
@@ -83,14 +75,22 @@
         }
     }
 
+    private final CatalogFramework catalogFramework;
+
+    private final FilterBuilder filterBuilder;
+
+    private String[] errorOnDuplicateAttributes;
+
+    private String[] warnOnDuplicateAttributes;
+
     public DuplicationValidator(CatalogFramework catalogFramework, FilterBuilder filterBuilder) {
         this.catalogFramework = catalogFramework;
         this.filterBuilder = filterBuilder;
     }
 
     /**
      * Setter for the list of attributes to test for duplication in the local catalog.  Resulting
-     * attributes will cause the {@link ddf.catalog.data.impl.BasicTypes#VALIDATION_ERRORS} attribute
+     * attributes will cause the {@link Validation#VALIDATION_ERRORS} attribute
```

Comment:
Either change this from `@link` to `@code` or fully qualify the reference to the `Validation` class. You can't import the class because it's not used but in javadoc in this class (and our checkstyle rules need to be tweaked to support that), but you can't leave it unqualified.